app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -39,7 +39,7 @@ def view_toolbar_filename
       'compare_view_tb'
     elsif @lastaction == "drift"
       'drift_view_tb'
-    elsif %w(ems_container).include?(@layout)
+    elsif %w(ems_container).include?(@layout) && %(timeline utilization).exclude?(@display)
```

Comment:
> summary + dashboard + topology and only for ems_container

does
`&& %(summary dashboard topology).include?(@display)`
do the trick ?

if it does, I think it's prettier, because i can more easily understand when this happen.